main.py

```diff
@@ -0,0 +1,19 @@
+import json
+from ui import UI
+
+
+# flash_cards() tries to read from a previous file "words_to_learn". If found calls UI class with previous
+# saved data. If not found, opens a fresh data of French words to learn
+def flash_cards():
+    try:
```

Comment:
You'll learn that I'm not a big fan of try / except's.  It tends to swallow big errors so I generally avoid unless there's some compelling reason.